### DISCUSSION TITLE
You'll Really Want to README ✨ [Profile Check-in]

### DISCUSSION THREAD

**Discussion by ebndev** [community/community#136395] [Other Feature Feedback, Questions, & Ideas]
![Header for the Profile category shows a solid, white background with green text reading “Profile August 2024 Check-in”. Underneath is an outline of the headings in this discussion: Community Highlights, GitHub Profile Updates, and Contribution Graphs. Next is a green banner with white text that reads “Thank you for all of your contributions this month!”. On the left side, there is the GitHub logo in the corner. Next to it is a green Octocat holding a paintbrush, wearing a black t-shirt, white and black pants, and has 4 cat claws as feet.](https://github.com/user-attachments/assets/7f6469ea-8522-4422-8680-d06fff8ffe6f)

Hello Community! 👋 

Welcome to our first check-in for this category! 🎉 Let's take a moment to celebrate all of your fantastic contributions and go through some updates to GitHub Profiles.

### ✨ Community Highlights

Check out some submissions from our [Showcase Your GitHub Profile! 🌟](https://github.com/orgs/community/discussions/121833) thread:
- See @tara-pogancev's very cool [all red](https://github.com/orgs/community/discussions/121833#discussioncomment-9578323) themed profile ❤️‍🔥
- Kudos to @imran-prog who uses his profile as a [portfolio and a CV](https://github.com/orgs/community/discussions/121833#discussioncomment-9670253) 🕴️
- Leave any useful tips + tricks for @pblnahu1, as he is [looking for recommendations](https://github.com/orgs/community/discussions/121833#discussioncomment-9716537) ✍️

Thank you to all 30+ users who've shared so far and commented kind, thoughtful feedback for others. If you haven't yet, please post yours in the thread and leave a reply in this discussion so we can check it out.

### 🖼️ GitHub Profile Updates
We’ve made a few important updates to GitHub profiles:

![Achievement badges with "Beta" badge and Send Feedback link](https://github.com/user-attachments/assets/e977fe94-a6b2-46e9-9e05-9dc14f2e4468)

- **Achievements Feedback:** We removed the "Beta" badge and feedback link from the Achievements section on the user profile page as Achievements isn't a "Beta" feature anymore. This link was causing user confusion and getting people to create posts in Community Discussions that were intended to be "feedback" about the user whose profile they were looking at. Although the link is no longer on user profiles, you can still provide feedback in the Profile category at any time! Check out these existing feature requests and comment / upvote / react on the discussions to show your support.
  - [Repo based achievements](https://github.com/orgs/community/discussions/75224)
  - [Re-organize position of achievements](https://github.com/orgs/community/discussions/125614)
  - [Remove achievements completely](https://github.com/orgs/community/discussions/115897)

- **GitHub Skyline:** We turned the lights off on the [Skyline microsite](https://skyline.github.com/) in June 2024. We may explore alternative ways to generate a 3D model of your contribution graph in the future but in the meantime, check out these awesome open-source projects built by members of our community!
    - [Skyline Website](https://git-skyline.huakun.tech/) by @huakunshen
    - [Skyline Repo](https://github.com/kamermans/github-skyline) by @kamermans
    - [Skyline Repo](https://github.com/avikalpg/skyline) by @avikalpg

- **Starring Repositories:** Following an incident on April 24th, we made the decision to remove starring repositories as a way to get contributions in contribution graphs. Unfortunately our investigation found this feature has recently started impacting our platform's availability, and implementing a safer version of it that behaves consistently for all users is surprisingly nuanced and difficult. If you have any additional thoughts or comments about this, please leave them in [this discussion](https://github.com/orgs/community/discussions/128895).

### 🟩 Contribution Graphs
Speaking of contribution graphs, here are some great answers to commonly asked questions: 
- @AndreaGriffiths11 explains [how colors of contribution graphs are determined](https://github.com/orgs/community/discussions/23261#discussioncomment-3239758)

- Wondering why your commits aren't showing? @davevad93 has provided many detailed solutions in [these discussions](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+involves%3Adavevad93+commits)

- Thanks @aaarajuan for this cool tip on how to [add this type of contribution graph to your profile's README!](https://github.com/orgs/community/discussions/62739#discussioncomment-6659126)
  ![ebndevs's GitHub activity graph](https://github-readme-activity-graph.vercel.app/graph?username=ebndev&theme=github-compact&title_color=51DD5DFF&hide_border=true)

As you all continue to learn and grow in your coding journeys, thanks for being a part of the GitHub Community!

**Comment by tara-pogancev** [comment]
Thank you for the shout out! 😄 It was so nice to see many amazing profile showcases and creative approaches, make sure to check the post out if you haven't already! 🥳

**Reply by avikalpg** [reply]
Thanks for the shoutout! I feel motivated to continue working on the repo and resolving the open issues 😇
